tests/test1.in

```diff
@@ -33,12 +33,12 @@ end
 # Simple alias tests
 
 function foo
-    echo >foo.txt $argv
+    echo >/tmp/fish_foo.txt $argv
```

Comment:
I decided to make the change to implement a *test/* directory tree before going to bed. It passes on my OS X and Ubuntu servers as well as two invocations on Travis CI and `make distclean` does the expected cleanups.